### PR TITLE
Bug/hide projects not supported on linux

### PR DESCRIPTION
**Pull request by DavidDamke**
The option Hide Projects on mouse leave does not work on Linux. Therefore it is now disabled

### CODE REVIEW

File: src/main/java/de/doubleslash/keeptime/view/SettingsController.java [hunk 1]
```diff
@@ -190,16 +191,6 @@ private void initialize() {
 
                alert.showAndWait();
             }
-            if (!displayProjectsRightCheckBox.isSelected() && hideProjectsOnMouseExitCheckBox.isSelected()) {
```
Comment (Death111) [inline src/main/java/de/doubleslash/keeptime/view/SettingsController.java:193]: can you add logic here, to set the value of hideProjects to false, if it is true. Maybe some users of a previous version have it active. We need to deactivate it then on settings save :)

Comment (DavidDamke) [reply src/main/java/de/doubleslash/keeptime/view/SettingsController.java:193]: Option is disabled and on save it get set to false 👍

File: src/main/java/de/doubleslash/keeptime/view/SettingsController.java [hunk 2]
```diff
@@ -190,15 +191,9 @@ private void initialize() {
...
+            if(hideProjectsOnMouseExitCheckBox.isSelected()){
                hideProjectsOnMouseExitCheckBox.setSelected(false);
-               final Alert warning = new Alert(AlertType.WARNING);
-               warning.setTitle("Warning!");
-               warning.setHeaderText("No Linux Support");
-               warning.getDialogPane().setMinHeight(Region.USE_PREF_SIZE);
-               warning.setContentText(
-                     "The project list on the left side has no Linux support if projects should be hidden. Disabling hiding of project list.");
-               warning.showAndWait();
+               hideProjectsOnMouseExitCheckBox.setDisable(true);
```
Comment (Death111) [inline src/main/java/de/doubleslash/keeptime/view/SettingsController.java:196]: we don't need to disable it (again)? it already is disabled from the start?

Comment (Death111) [reply src/main/java/de/doubleslash/keeptime/view/SettingsController.java]: removed the line

**Review by Death111 — APPROVED** [review]

**Pull request merged**